src/Controller/EnMarche/DonationController.php

```diff
@@ -129,11 +130,14 @@ public function resultAction(Request $request, Donation $donation)
             );
         }
 
+        $this->get(MembershipRegistrationProcess::class)->terminate();
```

Comment:
You should inject this service instead get it with container. With time more and more services will be put in private.